hms-api/src/error/ValidationError.ts

```diff
@@ -4,7 +4,7 @@ import ValidationResult from './ValidationResult';
 
 class ValidationError extends Error {
   constructor(message: string, result: ValidationResult) {
-    super(message + ' ' + result.toBulletList());
```

Comment:
was it making double spaces?

Reply:
yes it always added one extra space to the "message"